public/js/clients/firefox.js

```diff
@@ -16,13 +16,26 @@ function getTabTarget(tab) {
   return TargetFactory.forRemoteTab(options);
 }
 
+function presentTabs(tabs) {
```

Comment:
agreed. i think formatting is nicer, but I think there's a better word too. something closer to what an adapter would do. I'm thinking of the layer between sql and orm models, or orm models and UI views... I'll be doing lots of this work soon so will punt for the time being as well. not ignoring this though :P